### ISSUE TITLE
fresh-server-event-handlers warn happens in subfolder of islands

### ISSUE THREAD

**Issue by NyaightHazard** [denoland/deno_lint#1386]
### Lint Name

fresh-server-event-handlers

### Code Snippet

at `/islands/test/test.ts`:
```tsx
<button onClick={()=>{
    console.log("This is a test message.");
}}>Say something</button>
```

### Expected Result

No lint warning (OR dosn't work on fresh)

### Actual Result

Get lint warning but it works on fresh.

### Version

Deno 2.1.5
Fresh 1.7.3
